### PR TITLE
Routes cities #30

### PR DESCRIPTION
**Pull request by csjiang**

### CODE REVIEW

File: server/cities.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+'use strict'
+
+const db = require('APP/db')
+const City = db.model('city')
+
+const {forbidden,} = require('./auth.filters')
```
Comment (tessaSAC) [inline server/cities.js]: is this a typo?

File: server/test/cities.test.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+    it('DELETE deletes a city', () =>
+      request(app)
+        .delete('/api/cities/1')
+        .then(res => {
+          expect(res.body).to.equal(1)
+        })
+    )
+  })
+})
```
Comment (tessaSAC) [inline server/test/cities.test.js:61]: Do your db tests return expected results for broken tests?

Comment (csjiang) [reply server/test/cities.test.js:61]: What do you mean by "broken tests?"

**Review by antancia — APPROVED** [review]
Reviewed and accepted typo fix

File: server/cities.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+'use strict'
+
+const db = require('APP/db')
+const City = db.model('city')
+
+const {forbidden} = require('./auth.filters')
+
+module.exports = require('express').Router()
```
Comment (JolaWolak) [inline server/cities.js:8]: Don't we need a route for getting city data "by name" here?

**Review by JolaWolak — DISMISSED** [review]
A route for single city retrieval from db

**Pull request merged**